### PR TITLE
Add waffling80 hotswap PCB

### PR DESCRIPTION
**Pull request by 4pplet**
## Description

Adding support for waffling80 hotswap, a open source h88/h87 TKL PCB: https://github.com/4pplet/waffling80 (hotswap version not yet added to readme)

## Types of Changes
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/4pplet/waffling80/rev_b_ansi/rules.mk [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+
+# Wildcard to allow APM32 MCU
+DFU_SUFFIX_ARGS = -p FFFF -v FFFF
+
```
Comment (waffle87) [inline keyboards/4pplet/waffling80/rev_b_ansi/rules.mk:16]:
```suggestion
# Wildcard to allow APM32 MCU
DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```

File: keyboards/4pplet/waffling80/rev_b_ansi/info.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "keyboard_name": "waffling80 Rev B ANSI HS",
+    "usb": {
+        "pid": "0x0010",
+        "device_version": "0.0.1"
+    },
+    "rgblight": {
+        "saturation_steps": 8,
+        "brightness_steps": 8,
+        "led_count": 2
+    },
```
Comment (waffle87) [inline keyboards/4pplet/waffling80/rev_b_ansi/info.json:11]:
```suggestion
    },
    "features": {
        "bootmagic": true,
        "extrakey": true,
        "mousekey": true,
        "nkro": true,
        "rgblight": true
    }
```

**Comment by 4pplet** [comment]
Any updates on this? I believe it should be ready for merge

**Review by drashna — APPROVED** [review]

**Comment by 4pplet** [comment]
This should be ready for merge?

**Review by fauxpark — APPROVED** [review]

**Pull request merged**